amp/TEMPLATE/reamp/modules/activity/preview/containers/App.jsx

```diff
@@ -54,31 +51,34 @@ class App extends Component {
             DateUtils,
             activityFundingTotals,
             getAmountsInThousandsMessage: NumberUtils.getAmountsInThousandsMessage,
-            rawNumberToFormattedString:NumberUtils.rawNumberToFormattedString,
-            getActivityContactIds:ContactActions.getActivityContactIds,
+            rawNumberToFormattedString: NumberUtils.rawNumberToFormattedString,
+            getActivityContactIds: ContactActions.getActivityContactIds,
             contactsByIds
         }
     }
 
     render() {
-        return (
-            <ActivityPreviewUI activity={activity} activityContext={activityContext} />
-        );
+        console.log(this.props);
+        if (this.props.activityReducer.isActivityLoading) {
+            return (<div> LOADING ACTIVITY </div>)
```

Comment:
In a future phase we will add translate here.

Reply:
yes once we implement the translate module we will. Good catch